tests/functional/aws-node-sdk/test/multipleBackend/mpuParts/azurePutPart.js

```diff
@@ -0,0 +1,218 @@
+                    });
+                }, err => {
+                    assert.equal(err, null, 'Expected success, ' +
+                    `got error: ${err}`);
+                    checkSubPart(this.test.key, this.test.uploadId,
+                    parts, done);
+                });
+            });
+
+            it('should put 5 parts smaller than maxSubPartSize to Azure',
+            function it(done) {
+                const body = Buffer.alloc(10);
+                let parts = [];
+                for (let i = 1; i < 6; i++) {
+                    parts = parts.concat([
+                      { partnbr: i, subpartnbr: 0 },
+                    ]);
+                }
+                async.times(5, (n, next) => {
+                    const partNumber = n + 1;
+                    const params = {
+                        Bucket: azureContainerName,
+                        Key: this.test.key,
+                        UploadId: this.test.uploadId,
+                        PartNumber: partNumber,
+                        Body: body,
+                    };
+                    s3.uploadPart(params, (err, res) => {
+                        const eTagExpected = expectedETag(body);
+                        assert.strictEqual(res.ETag, eTagExpected);
+                        return next(err);
+                    });
+                }, err => {
+                    assert.equal(err, null, 'Expected success, ' +
+                    `got error: ${err}`);
+                    checkSubPart(this.test.key, this.test.uploadId,
+                    parts, done);
+                });
+            });
+
+            it('should put the same part twice', function itFn(done) {
+                const body1 = Buffer.alloc(maxSubPartSize + 10);
+                const body2 = Buffer.alloc(20);
+                const parts2 = [{ partnbr: 1, subpartnbr: 0 },
+                  { partnbr: 1, subpartnbr: 1 }];
+                async.waterfall([
+                    next => s3.uploadPart({
+                        Bucket: azureContainerName,
+                        Key: this.test.key,
+                        UploadId: this.test.uploadId,
+                        PartNumber: 1,
+                        Body: body1,
+                    }, err => next(err)),
+                    next => s3.uploadPart({
+                        Bucket: azureContainerName,
+                        Key: this.test.key,
+                        UploadId: this.test.uploadId,
+                        PartNumber: 1,
+                        Body: body2,
+                    }, (err, res) => {
+                        const eTagExpected = expectedETag(body2);
+                        assert.strictEqual(res.ETag, eTagExpected);
+                        return next(err);
+                    }),
+                    next => checkSubPart(this.test.key, this.test.uploadId,
+                    parts2, next),
+                ], done);
+            });
```

Comment:
need to test what happens when there was already a key with the same name as the mpu key.

Reply:
still need to test this with pre-existing key to make sure we are not overwriting.  you can use some of the setup stuff you have for the abort test but need this for put part as well.